pkgs/top-level/aliases.nix

```diff
@@ -67,6 +67,7 @@ mapAliases ({
   alsaUtils = alsa-utils; # Added 2021-06-10
   angelfish = libsForQt5.kdeGear.angelfish; # Added 2021-10-06
   ansible_2_12 = throw "Ansible 2.12 goes end of life in 2023/05 and can't be supported throughout the 23.05 release cycle"; # Added 2023-05-16
+  apacheAnt_1_9 = throw "Ant 1.9 has been removed since it's not used in nixpkgs anymore"; # Added 2023-11-12
```

Comment:
I don't know if it makes sense but couldn't it alias to apacheAnt instead? 🤔

Reply:
We can do that but it's weird to alias ant 1.9 to ant 1.10. Did we do that in other cases? If it's the convention I have no problem to follow it.

Reply:
Hmm I don't know what convention is on the matter. It is just a thought from a UX perspective to avoid unnecessary throws when users upgrade their systems.

Reply:
I thought it's better to throw instead of alias to another version and maybe breaks something else later.